ruma-events/src/lib.rs

```diff
@@ -256,6 +288,37 @@ impl RedactedSyncUnsigned {
     pub fn is_empty(&self) -> bool {
         self.redacted_because.is_none()
     }
+
+    /// Convert a `RedactedSyncUnsigned` into `RedactedUnsigned`, converting the
+    /// underlying sync redaction event to a full redaction event (with room_id).
+    pub fn into_full_event(self, room_id: RoomId) -> RedactedUnsigned {
```

Comment:
Since this is not an event itself, this method name is misleading. Maybe all the `into_full_event`s should just be named `into_full`? Or sth. like `with_room_id`.